assets/app/scripts/services/applicationGenerator.js

```diff
@@ -122,7 +122,7 @@ angular.module("openshiftConsole")
       templateLabels.deploymentconfig = input.name;
 
       var deploymentConfig = {
-        apiVersion: osApiVersion,
+        apiVersion: oApiVersion,
         kind: "DeploymentConfig",
```

Comment:
check the case on the trigger type values... I think the first letter is now upper-case (e.g. "Generic")